src/integrationTest/java/uk/gov/hmcts/divorce/citizen/notification/conditionalorder/AppliedForConditionalOrderNotificationIT.java

```diff
@@ -19,7 +19,7 @@
 import static uk.gov.hmcts.divorce.citizen.notification.conditionalorder.AppliedForConditionalOrderNotification.APPLICANT2;
 import static uk.gov.hmcts.divorce.citizen.notification.conditionalorder.AppliedForConditionalOrderNotification.PLUS_14_DUE_DATE;
 import static uk.gov.hmcts.divorce.divorcecase.model.Gender.FEMALE;
-import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.CONTACT_DIVORCE_JUSTICE_GOV_UK;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.CONTACT_DIVORCE_EMAIL;
```

Comment:
constant name can rename same ? will save changes in lot of file and name of the constant is relevant even though value has changed

Reply:
discussed on slack